server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJmapJamesServerTest.java

```diff
@@ -39,15 +40,15 @@
 import com.jayway.restassured.builder.RequestSpecBuilder;
 import com.jayway.restassured.http.ContentType;
 
-public abstract class AbstractJmapJamesServerTest {
+public abstract class AbstractJmapJamesServerTest<T extends GuiceJamesServer & JmapServer & WebAdminServer> {
```

Comment:
I understand your point but it seems a little overkill to me.
But if others agree, I'm fine with this.